Configure

```diff
@@ -1193,7 +1193,9 @@ else               { $no_user_defines=1;    }
 
 # ALL MODIFICATIONS TO %config and %target MUST BE DONE FROM HERE ON
 
-unless ($disabled{engine}) {
+if ($disabled{engine}) {
+    $disabled{afalg} = "no-engine";
+} else {
```

Comment:
> Ah yes, I was looking for something like this but saw it handled differently. I'll change that; but can you first explain the dependencies between dynamic-engine, static-engine and engine to me, so that I understand better what I'm actually doing?

`engine` regards the Engine API, so that's kind of the ring to rule them all when it comes to anything Engine.

`static-engine` and `dynamic-engine` are each other's opposite and they are about building dynamic engines such as `engines/capi.so` or have them statically included in `libcrypto`.  Internally, the only thing to care about is `$disabled{"dynamic-engine"}`, the config option `{no,enable,disable}-static-engine` is just for the user's convenience (or confusion) and gets translated into `$disabled{"dynamic-engine"}`.

Regarding your last question, there's the array `@disablables` where you might want to add this line:

``` perl
    "afalg",
```